ha_lineairdb.h

```diff
@@ -67,6 +67,29 @@ class LineairDB_share : public Handler_share {
   std::unique_ptr<LineairDB::Database> lineairdb_;
 };
 
+class Mysql_lineairdb_translator
+{
+public:
+  std::string nullFlags_;
+  std::string mysqlField_;
+private:
+  std::string lineairdbField_;
+
+  size_t read_header(const std::byte* const field, std::string& valBuf, size_t offset);
+  void encode_db_header(const size_t input);
+public:
+  std::string numeric_to_bytes (const size_t s);
+  template <typename BYTE_TYPE>
+  size_t bytes_to_numeric(const BYTE_TYPE* b, const size_t length);
+
+  template <typename BYTE_TYPE>
+  std::string&& encode_db_field(const BYTE_TYPE* srcMysql, const size_t length);
+
+  void set_null_flags_in_buf(uchar *buf);
+  size_t store_null_flags_to_mysql(const std::byte* const srcLineairdb, uchar *dstMysql);
+  size_t decode_db_field(const std::byte* const srcLineairdb, size_t offset);
```

Comment:
The names of the above methods are still ambiguous.
I want the name of the methods to clarify what `translator` is, even from outside this class.
(I think the current naming describes the work being done inside this class. It is not helpful)

For example, a method should describe what is input and what is output. `Translator` should have a method `A_to_B`, and it should be the only public method, isn’t it?

Reply:
I see that the method names are only describing the internal procedure and not what user should be expecting.
Users of these methods should be expecting to translate a MySQL field value to a LineairDB value entry, and vice versa, so `A_to_B` style of naming looks better here. Thank you for your suggestion.